tests/Checker/Checks/I18n_Usage_Check_Tests.php

```diff
@@ -0,0 +1,92 @@
+<?php
+/**
+ * Tests for the I18n_Usage_Check_Tests class.
+ *
+ * @package plugin-check
+ */
+
+use WordPress\Plugin_Check\Checker\Checks\I18n_Usage_Check;
+use WordPress\Plugin_Check\Checker\Check_Context;
+use WordPress\Plugin_Check\Checker\Check_Result;
+
+class I18n_Usage_Check_Tests extends WP_UnitTestCase {
+
+	protected $i18n_usage_check;
+
+	public function set_up() {
+		parent::set_up();
+
+		$this->i18n_usage_check = new I18n_Usage_Check();
+	}
+
+	/**
+	 * @covers I18n_Usage_Check::get_args()
+	 */
+	public function test_get_args() {
+
+		$sniffs = $this->i18n_usage_check->get_args();
+
+		$this->assertEquals(
+			array(
+				'extensions' => 'php',
+				'standard'   => 'WordPress',
+				'sniffs'     => 'WordPress.WP.I18n',
+			),
+			$sniffs
+		);
+	}
```

Comment:
See my above comment; if we make this method `protected`, we can't test it here anymore. That's fine though, since it's indirectly covered by the below tests, where it is called. If this method is broken, the tests below would fail.

Reply:
Correct, removed it.